topsort-api-v2.yml

```diff
@@ -82,6 +79,22 @@ paths:
                   maxItems: 10
               required:
                 - auctions
+              example:
+                - type: listings
+                  slots: 2
+                  products:
+                    - productId: p_OAmRYLx9qN51
+                  # category:
+                  #   id: c_yogurt
+                  # searchQuery: Noosa Peach
```

Comment:
If `slots` is 2, then it should have at least 2 products in the product list

Reply:
Good point.